
# HG changeset patch
# User hans@rhel5-devel.localdomain
# Date 1261672789 -3600
# Node ID 1cdd36e010cc5d9a7cc148076752727ad8e34343
# Parent  ee6f6ef50b2d79581801df739add9fe75a494499
libv4l: update upside down devices table

From: Hans de Goede <hdegoede@redhat.com>

libv4l: update upside down devices table

Priority: normal

Signed-off-by: Hans de Goede <hdegoede@redhat.com>

--- a/v4l2-apps/libv4l/libv4lconvert/control/libv4lcontrol.c	Thu Dec 24 07:37:56 2009 +0100
+++ b/v4l2-apps/libv4l/libv4lconvert/control/libv4lcontrol.c	Thu Dec 24 17:39:49 2009 +0100
@@ -137,9 +137,11 @@
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
   { 0x04f2, 0xb071, 0, "ASUSTeK Computer Inc.        ", "N61Vg     ",
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
+  { 0x04f2, 0xb071, 0, "ASUS CORPORATION            ", "N90SC     ",
+    V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
   { 0x04f2, 0xb071, 0, "ASUSTeK Computer Inc.        ", "U6Vc      ",
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
-  /* Note no whitespace padding for these 2 models, this is not a typo */
+  /* Note no whitespace padding for these models, this is not a typo */
   { 0x04f2, 0xb071, 0, "ASUSTeK Computer INC.", "K40AB",
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
   { 0x04f2, 0xb071, 0, "ASUSTeK Computer INC.", "K50AB",
@@ -177,6 +179,10 @@
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
   { 0x064e, 0xa116, 0, "ASUSTeK Computer Inc.        ", "K40IN      ",
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
+  { 0x064e, 0xa116, 0, "ASUSTeK Computer Inc.        ", "K60IJ     ",
+    V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
+  { 0x064e, 0xa116, 0, "ASUSTeK Computer Inc.        ", "K60IN      ",
+    V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
   { 0x064e, 0xa116, 0, "ASUSTeK Computer Inc.        ", "K70AB     ",
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
   { 0x064e, 0xa116, 0, "ASUSTeK Computer Inc.        ", "K70IO     ",

